aten/src/ATen/native/ReduceAllOps.cpp

```diff
@@ -11,21 +11,24 @@ DEFINE_DISPATCH(max_all_stub);
 DEFINE_DISPATCH(_aminmax_all_stub);
 
 Tensor min(const Tensor &self) {
-  TORCH_CHECK(self.numel() > 0, "operation does not have an identity.");
+  TORCH_CHECK(self.numel() > 0,
+              "min(): Expected reduction dim for self.numel() == 0. Specify the reduction dim with the 'dim' argument.");
```

Comment:
Prefer the user-facing names for errors messages, so "self.numel()" -> "input.numel()"

Reply:
Alright.